dali/python/CMakeLists.txt

```diff
@@ -15,18 +15,16 @@
 # Get all the srcs
 collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 collect_sources(DALI_PYTHON_BACKEND_SRCS)
-set(dali_python_lib "backend_impl")
-set(dali_python_lib ${dali_python_lib} PARENT_SCOPE)
 pybind11_add_module(${dali_python_lib} MODULE ${DALI_PYTHON_BACKEND_SRCS})
-message(STATUS "Adding dependencies to ${dali_python_lib}: '${dali_lib}'")
-add_dependencies(${dali_python_lib} ${dali_lib})
 
-# Link libdali.so, libprotobuf.a and libcudart_static.a into backend_impl (dali_python_lib)
-target_link_libraries(${dali_python_lib} PRIVATE ${CUDA_LIBRARIES} ${dali_lib})
+target_link_libraries(${dali_python_lib} PUBLIC
```

Comment:
Doesn't this make all of the DALI symbols visible in the lib? Previously it was only the python bindings.

Reply:
no, This only propagates dependencies so that if you now link against dali_python_lib you get its dependencies as well in the link flags (at least this is how I understood it)

Reply:
Ok.